README.md

```diff
@@ -65,6 +65,8 @@ with the following command:
 conda install -c conda-forge kitty-themes
 ```
 
+for those new to conda, the theme files can then be found at ~/opt/{anaconda3|miniconda3}/share/kitty-themes on macOS or ~/{anaconda3|miniconda3}/share on linux
```

Comment:
I think that the detail of where `anaconda` is installed should not "leak" in the documentation. 

Something along these lines should be enough:
You can find the themes inside the directory `$ANACONDA_INSTALLATION_DIR/share/kitty-themes/`.

Reply:
updated with link to conda docs for the installation dir, which was the hardest thing to find